src/pks/flow/constitutive_relations/sources/pipe_drain_evaluator.hh

```diff
@@ -49,7 +54,11 @@ class PipeDrainEvaluator : public EvaluatorSecondaryMonotypeCV {
  double energ_loss_coeff_weir_; // weir 
  double energ_loss_coeff_subweir_; // submerged weir
  double energ_loss_coeff_orifice_; // orifice
- double drain_length_; // drain conduit length
+ double sink_source_coeff_; // coefficient that determines sink or source (when using same evaluator file for pipe or surface flow) 
+ bool pipe_flag, sw_flag, pipe_map_created, sw_map_created;
```

Comment:
Please add `_` to the class variables at the end of their names, i.e. `pipe_flag_` for consistency

Reply:
Done